recipes/openms/meta.yaml

```diff
@@ -0,0 +1,153 @@
+{% set name = "OpenMS" %}
+{% set version = "2.9.1" %}
+package:
+  name: {{ name|lower }}-meta
+  version: {{ version }}
+
+{% if 'dev' in version %}
+source:
+  git_url: https://github.com/OpenMS/OpenMS.git
+  git_rev: develop
```

Comment:
could we please remove this condition, it really isn't useful for conda-forge and just adds challenges to maintanability.

Reply:
Yes I think we can. It made it a bit easier to have nightly builds but once this is close to finished I can get rid of it.